### PR TITLE
Fix variable name and remove useless mandatory

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

**Review by SlothOfAnarchy — APPROVED** [review]

**Review by Scriptkiddi — CHANGES_REQUESTED** [review]
Let me have a look

File: templates/pool.conf.j2 [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ; The address on which to accept FastCGI requests.
 ; Valid syntaxes are: 'ip.add.re.ss:port', 'port', '/path/to/unix/socket'.
 ; This option is mandatory for each pool.
-listen = {{ php_fpm_pool.listen | mandatory }}
```
Comment (Scriptkiddi) [inline templates/pool.conf.j2:10]: Why do you want to remove the mandatory vars

File: templates/pool.conf.j2 [hunk 2]
```diff
@@ -105,14 +105,14 @@ pm.max_children = {{ php_fpm_pool.pm_max_children | mandatory }}
...
-    pm.min_spare_servers = {{ php_fpm_pool.pm_min_spare_servers | mandatory}}
+    pm.min_spare_servers = {{ php_fpm_pool.pm_min_spare_servers}}
 {% endif %}
 
 ; The desired maximum number of idle server processes.
 ; Used only when pm is set to dynamic.
 ; Also mandatory in this case.
 {% if php_fpm_pool.pm == "dynamic" %}
-    pm.max_spare_servers = {{ php_fpm_pool.max_spare_servers | mandatory}}
+    pm.max_spare_servers = {{ php_fpm_pool.pm_max_spare_servers}}
```
Comment (Scriptkiddi) [inline templates/pool.conf.j2:115]: please also update the readme

**Review by Scriptkiddi — CHANGES_REQUESTED** [review]

File: templates/pool.conf.j2 (hunk 2 above)
Comment (Scriptkiddi) [reply templates/pool.conf.j2:115]: nevermind its max_children wrong var

Comment (dasJ) [reply templates/pool.conf.j2:115]: I would if you had put it in

File: templates/pool.conf.j2 (hunk 1 above)
Comment (dasJ) [reply templates/pool.conf.j2:10]: Because its implicit and error messages don't show which mandatory variable is undefined

Comment (SlothOfAnarchy) [reply templates/pool.conf.j2:10]: They're only necessary in tasks

**Comment by dasJ** [comment]
@Scriptkiddi merge pls

**Comment by dasJ** [comment]
Also, it was no problem fixing your role. You're welcome, no thanks needed.

**Review by Scriptkiddi — APPROVED** [review]

**Comment by Scriptkiddi** [comment]
Thanks for fixing it.

**Pull request merged**